debian/build.py

```diff
@@ -40,20 +41,18 @@ def __str__(self):
         self.pkg, self.arch, self.distro, self.version, self.revision)
 
 def main():
-  architectures=('amd64', 'arm', 'arm64')
+  architectures=('amd64',)
```

Comment:
Yes, I will make sure to remove it. This is just WIP. I'd be all up for figuring out how we can handle builds for debs and rpms in a way that would provide a single interface... I'm looking into fpm.